extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -168,6 +169,55 @@ private function getTitleFromRequest(){
 	}
 
 	/**
+	 * @desc Returns resource type for URI
+	 *
+	 * @throws BadRequestApiException
+	 */
+	public function getResourceType() {
```

Comment:
How about setResourceType? I guess would be more natural since getters usually return a value and this method is applying a value to the response object :dancer: